interface/main/messages/messages.php

```diff
@@ -1,668 +1,848 @@
-    echo ">" . htmlspecialchars('-Patient-', ENT_NOQUOTES);
-    echo "</option>\n";
-}
-?>
-   </select>
-  </td>
-   <td>
-    <input type ="button" name="clear_user" class= "css_button" id = "clear_user" value="<?php echo htmlspecialchars(xl('Clear'), ENT_QUOTES); ?>" >
-</td>
- </tr>
-
-<?php
-if ($noteid) {
-  // Get the related document IDs if any.
-    $tmp = sqlStatement(
-        "SELECT id1 FROM gprelations WHERE " .
-        "type1 = ? AND type2 = ? AND id2 = ?",
-        array('1', '6', $noteid)
-    );
-    if (sqlNumRows($tmp)) {
-        echo " <tr>\n";
-        echo "  <td class='text'><b>";
-        echo xlt('Linked document') . ":</b>\n";
-        while ($gprow = sqlFetchArray($tmp)) {
-            $d = new Document($gprow['id1']);
-            $enc_list = sqlStatement("SELECT fe.encounter,fe.date,openemr_postcalendar_categories.pc_catname FROM form_encounter AS fe ".
-            " left join openemr_postcalendar_categories on fe.pc_catid=openemr_postcalendar_categories.pc_catid  WHERE fe.pid = ? order by fe.date desc", array($prow['pid']));
-            $str_dob = htmlspecialchars(xl("DOB") . ":" . $prow['DOB'] . " " . xl("Age") . ":" . getPatientAge($prow['DOB']));
-            $pname = $prow['fname']." ".$prow['lname'];
-            echo "<a href='javascript:void(0);' ";
-            echo "onClick=\"gotoReport(".addslashes(attr($d->get_id())).",'".addslashes(attr($pname))."',".addslashes(attr($prow['pid'])).",".addslashes(attr($prow['pubpid'])).",'".addslashes(attr($str_dob))."');\">";
-            echo text($d->get_url_file());
-            echo "</a>\n";
+<html>
+<head>
+    <title><?php echo xlt('Message Center'); ?></title>
+
+    <?php Header::setupHeader(['jquery-ui', 'jquery-ui-redmond', 'opener', 'moment', 'pure']); ?>
+
+    <script>
+        <?php
+        if ($GLOBALS['date_display_format'] == '0') {
+            $date_format = 'yy-m-d';
+        } elseif ($GLOBALS['date_display_format'] == '1') {
+            $date_format = 'mm/dd/yy';
+        } elseif ($GLOBALS['date_display_format'] == '2') {
+            $date_format = 'dd/mm/yy';
         }
-
-        echo "  </td>\n";
-        echo " </tr>\n";
-    }
-
-  // Get the related procedure order IDs if any.
-    $tmp = sqlStatement(
-        "SELECT id1 FROM gprelations WHERE " .
-        "type1 = ? AND type2 = ? AND id2 = ?",
-        array('2', '6', $noteid)
-    );
-    if (sqlNumRows($tmp)) {
-        echo " <tr>\n";
-        echo "  <td class='text'><b>";
-        echo xlt('Linked procedure order') . ":</b>\n";
-        while ($gprow = sqlFetchArray($tmp)) {
-            echo "   <a href='";
-            echo $GLOBALS['webroot'] . "/interface/orders/single_order_results.php?orderid=";
-            echo $gprow['id1'];
-            echo "' target='_blank' onclick='top.restoreSession()'>";
-            echo $gprow['id1'];
-            echo "</a>\n";
+        ?>
+        var xljs_dateFormat = '<?php echo $date_format; ?>';
```

Comment:
TODO (for me)
Remove above date format code block after fully convert this PR to use standard date picker.